test/fixtures/data_exports.yml

```diff
@@ -0,0 +1,12 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+data_export_one:
+  name: Data Export 1
+  export_type: sample
+  status: processing
+  export_parameters: {sample_ids: [<%= ActiveRecord::FixtureSet.identify(:sample1) %>, <%= ActiveRecord::FixtureSet.identify(:sample) %>]}
```

Comment:
**nitpick (blocking):** Can you update `sample_ids` to just be `ids` as that is what you mention in your PR description.

Reply:
Fixed in 479f4ba8e2d8629ef98f5af57bd8ff0bc70fe5b8